plugins/aws_ec2/plugin.spec.yaml

```diff
@@ -758,6 +758,18 @@ connection:
     type: string
     example: "us-east-1"
     required: true
+  role_arn:
```

Comment:
Would it be best to place these in a single custom type that can be added to every action?

Reply:
Idk, on one hand it would slightly reduce amount of code, on the other hand it would make the ui and the plugin.spec.yaml more complex

Reply:
Anyhow, we will be refactoring AWS plugins so thats an open question on how to improve that process